framework/db/target_manager.py

```diff
@@ -343,8 +344,8 @@ def GetTargetsSeverityCount(self):
             {"id":5, "label": "High", "value": 0, "color": "#c12e2a"},
             {"id":6, "label": "Critical", "value": 0, "color": "#800080"}
         ]
-        total = self.db.session.query(models.Target).count()
-        target_objs = self.db.session.query(models.Target).all()
+        total = self.db.session.query(models.Target).filter(models.Target.sessions.any(id=session_id)).count()
+        target_objs = self.db.session.query(models.Target).filter(models.Target.sessions.any(id=session_id)).all()
```

Comment:
A better way to write this  should be 
`filter(models.Target.sessions.id=session_id).all()`. Right?